tests/swift/benchmarks/Benchmarks/FlatbuffersBenchmarks/FlatbuffersBenchmarks.swift

```diff
@@ -0,0 +1,154 @@
+///*
+// * Copyright 2023 Google Inc. All rights reserved.
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// */
+
+import Benchmark
+import CoreFoundation
+import FlatBuffers
+
+@usableFromInline
+struct AA: NativeStruct {
+  public init(a: Double, b: Double) {
+    self.a = a
+    self.b = b
+  }
+  var a: Double
+  var b: Double
+}
+
+let benchmarks = {
+  let ints: [Int] = Array(repeating: 42, count: 100)
+  let bytes: [UInt8] = Array(repeating: 42, count: 100)
+  let str = (0...99).map { _ -> String in "x" }.joined()
+  let array: [AA] = [
+    AA(a: 2.4, b: 2.4),
+    AA(a: 2.4, b: 2.4),
+    AA(a: 2.4, b: 2.4),
+    AA(a: 2.4, b: 2.4),
+    AA(a: 2.4, b: 2.4),
+  ]
+
+  Benchmark("Allocating") { benchmark in
+    for i in benchmark.scaledIterations {
+      var fb = FlatBufferBuilder(initialSize: 1_024_000_000)
+    }
+  }
+
+  Benchmark("Strings 10") { benchmark in
+    var fb = FlatBufferBuilder(initialSize: 1<<20)
+    for i in benchmark.scaledIterations {
+      benchmark.startMeasurement()
```

Comment:
You want to move this outside of the loop for scaledIterations - otherwise you'll only measure the last loop - same for stopMeasurement  - should update the docs to clarify.